docs/node-operators/run-evm-tracing-node.md

```diff
@@ -22,14 +22,14 @@ As you may be aware, Darwinia chains are EVM compatible, which means that the RP
     
 2. Download the precompiled tracing binary
     
-    As of the time of writing this doc (2024-09-02), the latest version of the Koi Chain is `v6.6.5`. Please ensure that you check for [the latest version](https://github.com/darwinia-network/darwinia/releases) when running your own node.
+    As of the time of writing this doc (2024-09-27), the latest version of the Koi Chain is `v6.7.1`. Please ensure that you check for [the latest version](https://github.com/darwinia-network/darwinia/releases) when running your own node.
 
     !!! warn 
         Please be aware that there are two types of binaries available on the release page. Select the one that starts with `darwinia-tracing-xxx`.
     
     ```bash
-    wget https://github.com/darwinia-network/darwinia/releases/download/v6.6.5/darwinia-x86_64-linux-gnu.tar.bz2
-    tar xvf darwinia-x86_64-linux-gnu.tar.bz2
+    wget https://github.com/darwinia-network/darwinia/releases/download/v6.7.1/darwinia-tracing-x86_64-linux-gnu.tar.zst
+    tar -I zstd -xf darwinia-tracing-x86_64-linux-gnu.tar.zst
```

Comment:
Please convert this to the standard Darwinia release package. For the EVM tracing node, we utilize the common Darwinia binary along with runtime override files.

Reply:
I'm confused there, the above tip use [darwinia-tracing-xxx](https://github.com/darwinia-network/docs/blob/87c8d7aedcca5e774dea5c29fa3c71ec39d47da6/docs/node-operators/run-evm-tracing-node.md?plain=1#L27-L28)

and our tracing node use tracing binary https://github.com/darwinia-network/devops/blob/d5a1931211195c59af7ac7e767f20d08ad6e5183/docker-image/darwinia-tracing/Dockerfile#L17

Reply:
Give me a second. I need to check it again.

Reply:
Sorry, you are right. Ignore this comment.